shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/analytics/Analytics.kt

```diff
@@ -161,6 +195,7 @@ abstract class Analytics {
     }
 
     fun track(screen: AnalyticsScreen) {
+        lastTrackedScreen = screen
```

Comment:
This seemed like the most straightforward way to avoid having to drill the context through to the lower-level route card component where the tracking actually happens.